build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'net.sf.kxml'
+version = '2.3.2'
```

Comment:
I think it should be at least 2.4.2, as I have already tagged the last bugfix as 2.4.1 (given the project move and time span -- perhaps a bit early): https://github.com/stefanhaustein/kxml2/releases

In my view, a 2.5.0. would be better for a new project layout.